surface.js

```diff
@@ -67,13 +67,14 @@ function SurfacePickResult (position, index, uv, level, dataCoordinate) {
 
 var N_COLORS = 256
 
-function genColormap (name) {
+function genColormap (name, opacityscale) {
```

Comment:
It would be nice to start a helper module in gl-vis to reuse logic like this in both gl-mesh3d and gl-surface3d.